lib/kitchen/metadata_element.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Kitchen
+  # An element for metadata
+  #
+  class MetadataElement < ElementBase
+    # Creates a new +MetadataElement+
+    #
+    # @param node [Nokogiri::XML::Node] the node this element wraps
+    # @param document [Document] this element's document
+    #
+    def initialize(node:, document: nil)
+      super(node: node,
+            document: document,
+            enumerator_class: MetadataElementEnumerator,
+            short_type: :metadata)
+    end
+
+    # Returns set of selected data elements
+    #
+    # @return [ElementEnumerator]
+    #
+    def data
```

Comment:
Is there a more meaningful name we can use?  Like `elements_to_keep`? (that name isn't awesome either but it conveys that these are child elements that we want to persist through baking.

Reply:
What about these?: `data_to_keep` / `children_to_keep` / `descendants_to_keep`